### ISSUE TITLE
Style Loader Invalid Options - options.importLoaders next@5

### ISSUE THREAD

**Issue by schoenwaldnils** [zeit/next.js#3727]
- [x] I have searched the [issues](https://github.com/zeit/next.js/issues) of this repository and believe that this is not a duplicate.

I get this error message on `next build`
```
Module build failed: ValidationError: Style Loader Invalid Options
options['importLoaders'] should NOT have additional properties
```

- [next.config.js](https://github.com/schoenwaldnils/blog/blob/greenkeeper/next-5.0.0/next.config.js)
- [.babelrc](https://github.com/schoenwaldnils/blog/blob/greenkeeper/next-5.0.0/.babelrc)
- [All the other files](https://github.com/schoenwaldnils/blog/tree/greenkeeper/next-5.0.0)

## Your Environment

| Tech    | Version |
|---------|---------|
| next    | 5.0.0, 5.0.1-canary.5     |
| node    |  8.9.1   |

**Comment by schoenwaldnils** [comment]
Still not fixed in `next@5.0.1-canary.5` and still blocking me from upgrading to next@5

**Comment by timneutkens** [comment]
Upgrade next-css next-sass and or next-less. The config was removed.

**Comment by timneutkens** [comment]
This is an issue for github.com/zeit/next-plugins btw.

**Comment by schoenwaldnils** [comment]
can confirm it's fixed in @zeit/next-css@0.1.2
